### PR TITLE
Create Github Actions workflow

### PR DESCRIPTION
**Pull request by atmaxinger**
Hi!

I thought this project was lacking a CI workflow so I created one. (Well at least no working one that is, it seems that the travis-ci workflow doesn't work)

It's currently very basic and only builds on Linux, but it's a start.

### CODE REVIEW

File: .github/workflows/sourcehold.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Sourcehold
+
+on: [push]
```
Comment (gynt) [inline .github/workflows/sourcehold.yml:3]: Maybe also add on pull request, so we can see whether PR's actually build!

**Review by gynt — CHANGES_REQUESTED** [review]

File: .github/workflows/sourcehold.yml (hunk 1 above)
Comment (atmaxinger) [reply .github/workflows/sourcehold.yml:3]: It does it automatically on PRs too - see my test PR here: https://github.com/atmaxinger/Sourcehold/pull/2

Comment (atmaxinger) [reply .github/workflows/sourcehold.yml]: Nevermind, explicitly stating it doesn't harm anyone. Done.

**Comment by atmaxinger** [comment]
Ok now with the pull_request trigger added it builds it twice. One time the original branch, and then the merged branches. 
I think that's pretty neat so we can see whether it's the merge or the original branch making troubles.

**Review by gynt — APPROVED** [review]

**Pull request merged**